### ISSUE TITLE
Can't reproduce the result using pytorch 0.4.1 branch

### ISSUE THREAD

**Issue by kebinC** [fartashf/vsepp#30]
Hi, i can't reproduce the result using pytorch 0.4.1 branch
the command：
```
python train.py \
    --data_path ./data \
    --data_name coco \
    --logger_name outputs/coco_vse++_vggfull_restval \
    --use_restval \
    --max_violation
```

the loss and R@1 curve is showed below.   The Le is higher than baseline  and R1 is lower than baseline.

![image](https://user-images.githubusercontent.com/14308158/89444859-f2f4b600-d784-11ea-85d3-4519df1d8f14.png)

![image](https://user-images.githubusercontent.com/14308158/89444877-fa1bc400-d784-11ea-9c25-86a5e4a9fe9c.png)

**Comment by fartashf** [comment]
Thanks for reporting. I did not get a chance to take a look. Can you comment on why the issue is closed?

**Comment by kebinC** [comment]
> Thanks for reporting. I did not get a chance to take a look. Can you comment on why the issue is closed?

The  reason why i can't reproduce the result is same as #4. i set  the cnn eval mode and fix this problem.

**Comment by fartashf** [comment]
Thanks for clarifying.
